source/core/GenericFunctions.cpp

```diff
@@ -1848,11 +1848,11 @@ VIREO_FUNCTION_SIGNATURET(ClusterBinaryOp, AggregateBinOpInstruction)
             int whichConvertArg = -int(intptr_t(convertSnippet->_p1));
             snippet = (Instruction4<AQBlock1, AQBlock1, AQBlock1, AQBlock1>*)convertSnippet->Next();
             if (whichConvertArg == 1) {
-                convertSnippet->_p0 = snippet->_p0 + (size_t)_ParamPointer(SX);
```

Comment:
snippet->_p0 corresponds to the offset in the actual instruction we're trying to perform (e.g. Sub or Remainder) not the conversion instruction. However, in [Generics.cpp#303](https://github.com/segaljared/VireoSDK/blob/3562cf61af93e76ce1b967cc45213b5ac630224b/source/core/GenericFunctions.cpp#L303), for example,  we've already put the destination offset in that parameter, so we'll end up using the wrong offset. The right offset has already been put into the conversion instruction, so we just need to add _ParamPointer(SX) like we would otherwise.